### PR TITLE
Revert "chore: no need to overwrite the selenium version"

### PR DESCRIPTION
**Pull request by Artur-**
Reverts vaadin/skeleton-starter-flow-spring#867

### CODE REVIEW

**Comment by ZheSun88** [comment]
why we need to revert this?
selenium version should come from TB, no?

**Comment by Artur-** [comment]
No, from Spring Boot

**Review by ZheSun88 — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -12,6 +12,7 @@
     <properties>
         <java.version>11</java.version>
         <vaadin.version>23.2.3</vaadin.version>
+        <!-- this parameter is needed as spring-boot bom overwrites it -->
```
Comment (Artur-) [inline pom.xml:15]: overwrites what?

Comment (Artur-) [reply pom.xml:15]: This is super confusing to me as a user

**Pull request merged**